docker/docker-compose.yml

```diff
@@ -251,7 +251,7 @@ services:
     restart: unless-stopped
     environment:
       JWT_SECRET: ${JWT_SECRET}
-      SUPABASE_URL: http://kong:8000
+      SUPABASE_URL: ${SUPABASE_PUBLIC_URL}
```

Comment:
This is probably not what you want. Inside of the functions container this URL is used to create a supabase client, which -since it is part of the same docker setup - should probably call the kong container directly, i.e. http://kong:8000

Reply:
I just tested again and looks fine now. There might be something else wrong in the previous version of the self-host that I changed.